engine/src/main/java/org/terasology/rendering/logic/LightComponent.java

```diff
@@ -43,14 +44,29 @@
     @Replicate
     public float lightAmbientIntensity = 1.0f;
 
+    /**
+     * This helps control how focused the specular light is. A smaller number will make a wider cone of light. A larger number will make a narrower cone of light.
+     */
     @Replicate
-    public float lightSpecularPower = 1.0f;
```

Comment:
Oh,  sorry I was not as clear as I could have been.  These images are with specular light intensity at 1.0 and the other intensities at 0.0.   It is only the specular component of the light.